lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_facts.py

```diff
@@ -254,7 +254,7 @@ def __init__(self):
 
         self.results = dict(
             changed=False,
-            vms=[]
+            ansible_facts=dict(azure_virtualmachine=[])
```

Comment:
No cloud modules should return `ansible_facts`. All cloud `_facts` modules should be `_info` modules.